tools/topology/topology1/m4/google_ctc_audio_processing_coef_default.m4

```diff
@@ -0,0 +1,10 @@
+CONTROLBYTES_PRIV(DEF_CTC_PRIV,
+`       bytes "0x53,0x4f,0x46,0x34,0x00,0x00,0x00,0x00,'
+`       0x18,0x10,0x00,0x00,0x00,0xD0,0x01,0x03,'
```

Comment:
`0x18,0x10,0x00,0x00` this int32 is config size and apparently wrong. I'm not sure if it may do something bad to module since it may grab data outside of your definition